CMake/cdat_modules_extra/git_clone.sh.in

```diff
@@ -3,7 +3,7 @@
 cd "@CMAKE_INSTALL_PREFIX@"
 "@GIT_EXECUTABLE@" clone --no-checkout --depth 1 -b @BRANCH@ @GIT_URL@ "@GIT_TARGET@"
```

Comment:
It looks like --depth cannot be used if we get rid of -b, else git only gets HEAD and the ref we need might not be fetched at all.